src/core/components/NavBar.module.css

```diff
@@ -70,18 +70,20 @@
 }
 
 .dropdownMenu {
-  position: absolute;
-  top: 100%;
+  position: fixed;
+  top: 62px;
```

Comment:
The dropdown menu wasn't actually clickable on mobile - if you selected an option, the click event was happening below the dropdown, triggering close on outside click, making switching on mobile impossible.